mithril-common/ADR.md

```diff
@@ -0,0 +1,61 @@
+# Mithril Common Architecture Decision Records
+
+## 1. Record Architecture Decisions
+
+date: 2025-02-26
+status: Accepted
```

Comment:
Is it really useful to have a status ? 
Will there ever be anything other than `Accepted` ?

Reply:
I agree that it have a low value, the only one that I see is when an entry supersede a previous one as it allow us to mark the previous entry with a link to the new one.
I will keep it for now, we may remove them in the future if this case can be handled differently.